C/other/Longest substring without repeating chars.cpp

```diff
@@ -0,0 +1,77 @@
+//This code is submitted to NeoAlgo as a part of Contributors hack 2020
+
+//Finding the longest substring without repeating characters having linear time Complexity O(n) and constant Space Complexity O(1)
+
+------------------------------------------------------------------------------
+	
+//Sample Input-output
+------------------------------------------------------------------------------
+//Input: abcbacbb
+//Output: length of longest substring without repeating charecters: 3
+
+//Input: Hackincodes
+//Output: length of longest substring without repeating charecters: 7
+
+//Input: ddddd
+//Output: length of longest substring without repeating charecters: 1
+
+//Input: 
+//Output: length of longest substring without repeating charecters: 0
+------------------------------------------------------------------------------

```

Comment:
@Yuvraj-kadale Remove these type of line patterns.

Reply:
Done removed them!!